src/physics/game_states/game_states.py

```diff
@@ -0,0 +1,10 @@
+from util.io.game_state import GameState
+from util.state_machine.state_machine import State
+
+
+class InitGameLogic(State):
+    def exec(self, param: GameState):
+        return None
+
+    def next(self, param) -> State:
+        return self

```

Comment:
this state is for initializing a game ONLY when the 2 clients and server are connected. It's the init of the GAME after the init of the NETWORK